nodes/text/stethoscope_v28.py

```diff
@@ -34,6 +34,22 @@
 FAIL_COLOR = (0.1, 0.05, 0)
 READY_COLOR = (1, 0.3, 0)
 
+
+def adjust_location(_x, _y, location_theta):
+    return _x * location_theta, _y * location_theta
+
+def get_xy_for_bgl_drawing(node):
+        # adjust proposed text location in case node is framed.
+        # take into consideration the hidden state
+        node_width = node.width
+        _x, _y = node.absolute_location
+        _x, _y = Vector((_x, _y)) + Vector((node_width + 20, 0))
+
+        # this alters location based on DPI/Scale settings.
+        _, location_theta = node.get_preferences()
```

Comment:
this may be a performance bottleneck, and maybe it makes sense to store this (scale/location_theta) during the process event, rather than calling this function on every callback draw.